docs/_pages/releases.md

```diff
@@ -53,6 +53,7 @@ sidebar:
 * `OnlyContain` now succeeds when asserting that an empty collection matches some predicates - [#2350](https://github.com/fluentassertions/fluentassertions/pull/2350)
 * Dropped support for `NSpec3` test framework - [#2356](https://github.com/fluentassertions/fluentassertions/pull/2356)
 * Dropped support for `BinaryFormatter` - [#2278](https://github.com/fluentassertions/fluentassertions/pull/2278)
+* Renamed `EquivalencyAssertionOptions` to `EquivalencyOptions` - [#2414](https://github.com/fluentassertions/fluentassertions/pull/2414)
```

Comment:
Please mention all changes types

* `EquivalencyAssertionOptions` -> `EquivalencyOptions`
* `EquivalencyAssertionOptions<TExpectation>` -> `EquivalencyOptions<TExpectation>`
* `IEquivalencyAssertionOptions` -> `IEquivalencyOptions`
* `SelfReferenceEquivalencyAssertionOptions<TSelf>` -> `SelfReferenceEquivalencyOptions<TSelf>`

Reply:
I changed the `releases.md` file and rebased to latest develop.